buttons/export.md

```diff
@@ -132,6 +132,12 @@ class UsersDataTable extends DataTable
 ...
 ```
 
+And also add this code to your routes.php file.
+```php
+    $router->post('sample/export', SampleController::class . '@index');
```

Comment:
Maybe also include the route for the main dataTable index for clarity?
```php
Route::resource('sample', 'SampleController@index');
Route::post('sample/export', 'SampleController@index');
```

Reply:
Done changing routes instruction.